components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/staticnetwork/StaticMemberRegistrationService.kt

```diff
@@ -211,31 +214,30 @@ class StaticMemberRegistrationService @Activate constructor(
 
     private fun persistGroupParameters(memberInfo: MemberInfo, staticMemberList: List<StaticMember>) {
         val cache = lifecycleHandler.groupParametersCache
-        val groupParametersList = cache.getOrCreateGroupParameters(memberInfo.holdingIdentity).run {
+        val holdingIdentity = memberInfo.holdingIdentity
+        val groupParametersList = cache.getOrCreateGroupParameters(holdingIdentity).run {
             memberInfo.notaryDetails?.let {
                 cache.addNotary(memberInfo)
             } ?: this
         }
         val groupParameters = groupParametersFactory.create(groupParametersList)
 
-        // Persist group parameters for this member
-        persistenceClient.persistGroupParameters(
-            memberInfo.holdingIdentity,
-            groupParameters
-        )
+        // Persist group parameters for this member, and publish to Kafka for DB reconciliation.
```

Comment:
```suggestion
        // Persist group parameters for this member, and publish to Kafka.
```
No need to mention database reconciliation here I think.